cfgov/jinja2/v1/ask-cfpb/category-page.html

```diff
@@ -39,43 +39,40 @@
         </div>
     </section>
 
-    <section class="ask-categories">
+    {% if choices|length > 0 %}
+    <section class="ask-categories
+                    block block__bg 
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3033/commits/21c18afdc25776df21caf4cf90134ef9718838e6.